index.html

```diff
@@ -7,33 +7,60 @@
         <script type="text/javascript" src="./js/changeIcon.js" defer></script>
     </head>
     <body>
-        <header>
-            <div class="logo">
-                <a href="index.html">
-                    <img src="./svg/logo.svg" alt="Logo"/>
-                    <p>Apartment Owners Association</p>
-                </a>
-            </div>
-            <div class="navigation" id="navButtons">
-                <a href="#about">About us</a>
-                <a href="#membership">Membership</a>
-                <a href="#services">Services</a>
-                <a href="#news">News and events</a>
-                <a href="#contact">Contact</a>
-            </div>
-            <div class="mobileIconContainer" onclick="handleClickMobile(this)">
-                <div class="mobileNavBar1"></div>
-                <div class="mobileNavBar2"></div>
-                <div class="mobileNavBar3"></div>
-            </div>
-        </header>
-        <section class="hero">
-            <div class="container">
+        <div class="page">
+            <header>
+                <div class="logo">
+                    <a href="index.html">
+                        <img src="./svg/logo.svg" alt="Logo"/>
+                        <p>Apartment Owners Association</p>
+                    </a>
+                </div>
+                <div class="navigation" id="navButtons">
+                    <a href="#about">About us</a>
+                    <a href="#membership">Membership</a>
+                    <a href="#services">Services</a>
+                    <a href="#news">News and events</a>
+                    <a href="#contact">Contact</a>
+                </div>
+                <div class="mobileIconContainer" onclick="handleClickMobile(this)">
+                    <div class="mobileNavBar1"></div>
+                    <div class="mobileNavBar2"></div>
+                    <div class="mobileNavBar3"></div>
+                </div>
+            </header>
+            <section class="hero">
+                <div class="container">
                 <span>
```

Comment:
Avoid using inline element as a wrapper for block elements. In this case div works better.